diff -up ./include/fastcdr/config.h.in.endian ./include/fastcdr/config.h.in
--- ./include/fastcdr/config.h.in.endian	2017-06-10 09:54:12.654716780 -0400
+++ ./include/fastcdr/config.h.in	2017-06-10 09:54:42.102833882 -0400
@@ -44,8 +44,8 @@
 #endif
 
 // Endianness defines
-#ifndef __BIG_ENDIAN__
-#define __BIG_ENDIAN__ @__BIG_ENDIAN__@
+#ifndef FASTCDR_BIG_ENDIAN__
+#define FASTCDR_BIG_ENDIAN__ @__BIG_ENDIAN__@
 #endif
 
 // C++11 Non-copyable feature
diff -up ./src/cpp/Cdr.cpp.endian ./src/cpp/Cdr.cpp
--- ./src/cpp/Cdr.cpp.endian	2017-06-10 09:54:57.845431069 -0400
+++ ./src/cpp/Cdr.cpp	2017-06-10 09:55:09.890888008 -0400
@@ -18,7 +18,7 @@
 using namespace eprosima::fastcdr;
 using namespace ::exception;
 
-#if __BIG_ENDIAN__
+#if FASTCDR_BIG_ENDIAN__
     const Cdr::Endianness Cdr::DEFAULT_ENDIAN = BIG_ENDIANNESS;
 #else
     const Cdr::Endianness Cdr::DEFAULT_ENDIAN = LITTLE_ENDIANNESS;
